.circleci/config.yml

```diff
@@ -93,6 +93,9 @@ workflows:
 
 jobs:
   build:
+    branches:
+      only:
+        - master
```

Comment:
I don't think that this works. :)

The main issue is that CircleCI listens on webhooks incoming from GitHub and if there is no config available on the branch it simply fails (this is awkward, but okay).